lib/kitchen/figure_element.rb

```diff
@@ -20,7 +20,25 @@ def initialize(node:, document: nil)
     # @return [Element, nil]
     #
     def caption
-      first('figcaption')
+      first('figcaption', "div[data-type='description']")
+    end
+
+    # Returns true if the figure is a child of another figure
+    #
+    # @return [Boolean]
+    #
+
+    def is_subfigure?
+      parent.node.to_s.end_with?('</figure>')
```

Comment:
does `parent.name == 'figure'` work?

I know this is covered by `BakeFigure`'s spec, but might be nice to add a simple spec to `figure_element_spec.rb` to test this directly.